### ISSUE TITLE
Tweak brewTags on brewItem

### ISSUE THREAD

**Issue by ericscheid** [naturalcrit/homebrewery#2334]
Via #2318 
> it might be worth another PR to add a similar styling for each tag as it appears in the Metadata panel. Otherwise it can be hard to tell where one tag ends and another begins, especially with multi-word tags:

![image](https://user-images.githubusercontent.com/2605467/188363129-227ec85b-a6fc-4764-a05f-57a366367a9f.png)

**Comment by ericscheid** [comment]
<img width="425" alt="image" src="https://user-images.githubusercontent.com/2605467/188365612-6715942e-8d22-4a11-87bf-266377a64b35.png">

```html
<div class="brewTags">
<i class="fas fa-tags"></i>
  <span>Campaign</span>
  <span>D&amp;D 2e</span>
  <span>SpellJammer</span>
</div>
```

```css
.brewTags span {
  background-color: #c8ac6e3b;
  margin-left: 2px;
  border-radius: 4px;
  padding: 2px;
  border: 1px solid #c8ac6e;
  font-weight: 600;
}
```

and change this generator code:

https://github.com/naturalcrit/homebrewery/blob/8d10800cb69586dabdec86f83c40b697e577d005/client/homebrew/pages/basePages/listPage/brewItem/brewItem.jsx#L109

**Comment by calculuschild** [comment]
Fixed by #2344
